addons/laser_selfdesignate/stringtable.xml

```diff
@@ -4,9 +4,11 @@
   <Package name="Laser_SelfDesignate">
     <Key ID="STR_ACE_Laser_SelfDesignate_DesignatorOn">
       <English>Laser&lt;br/&gt;Designator On</English>
+      <Polish>Laser&lt;br/&gt;Dezygnator wł.</Polish>
```

Comment:
Dezygnator? To w ogóle coś takiego istnieje? Pewnie literówka, więc powinno być Desygnator choć to tego też nie jestem w 100% pewien. Może wskaźnik?